src/main/kotlin/net/maizegenetics/phgv2/brapi/model/DataFile.kt

```diff
@@ -0,0 +1,30 @@
+package model
+
+import kotlinx.serialization.Serializable
+
+/**
+ * A dataFile contains a URL and the relevant file metadata to represent a file
+ * @param fileDescription A human readable description of the file contents
+ * @param fileMD5Hash The MD5 Hash of the file contents to be used as a check sum
+ * @param fileName The name of the file
+ * @param fileSize The size of the file in bytes
+ * @param fileType The type or format of the file. Preferably MIME Type.
+ * @param fileURL The absolute URL where the file is located
+ */
+@Serializable
+data class DataFile (
+        /* The absolute URL where the file is located */
+        val fileURL: kotlin.String
```

Comment:
Can we remove the kotlin. bits?  This looks like a weird Java -> kotlin conversion.

Reply:
done